### PR TITLE
Route Reflector feature

### PR DESCRIPTION
**Pull request by ut0mt8**
As discussed in #192 please find the PR enabling Route Reflector feature.

### CODE REVIEW

File: app/controllers/network_routes_controller.go [hunk 1]
```diff
@@ -905,18 +911,10 @@ func (nrc *NetworkRoutingController) disableSourceDestinationCheck() {
 			},
 		)
 		if err != nil {
-			awserr := err.(awserr.Error)
-			if awserr.Code() == "UnauthorizedOperation" {
```
Comment (murali-reddy) [inline app/controllers/network_routes_controller.go]: Any reason why this piece of code is removed?

File: app/controllers/network_routes_controller.go [hunk 2]
```diff
@@ -905,18 +911,10 @@ func (nrc *NetworkRoutingController) disableSourceDestinationCheck() {
...
-				glog.Errorf("Node does not have necessary IAM creds to modify instance attribute. So skipping disabling src-dst check.")
-				return
-			}
-			glog.Errorf("Failed to disable source destination check due to: %v", err.Error())
+			glog.Errorf("Failed to disable source destination check due to: " + err.Error())
 		} else {
 			glog.Infof("Disabled source destination check for the instance: " + instanceID)
 		}
-
-		// to prevent EC2 rejecting API call due to API throttling give a delay between the calls
```
Comment (murali-reddy) [inline app/controllers/network_routes_controller.go]: Same here. Intentionally deleted?

**Comment by ut0mt8** [comment]
Absolutely not. I can retry applying my patch on a clean branch and remake a PR if you want.

**Comment by murali-reddy** [comment]
> Absolutely not. I can retry applying my patch on a clean branch and remake a PR if you want.

@ut0mt8 Please, do. Otherwise changes look good. I will test it out tommrow. 

Thanks for the patch BTW :)

**Pull request closed without merging**